### ISSUE TITLE
slidy create version solving failed

### ISSUE THREAD

**Issue by FilipKvestak** [Flutterando/slidy#213]
Cannot get packages after creating new project with slidy:

```
[mobx_counter] flutter pub get
Running "flutter pub get" in mobx_counter...                    
The current Dart SDK version is 2.10.4.

Because modular_codegen >=2.1.0 <3.0.0-nullsafety.1 depends on flutter_modular_annotations >=0.0.2 which requires SDK version >=2.12.0-0 <3.0.0, modular_codegen >=2.1.0 <3.0.0-nullsafety.1 is forbidden.

So, because mobx_counter depends on modular_codegen ^2.1.0, version solving failed.
pub get failed (1; So, because mobx_counter depends on modular_codegen ^2.1.0, version solving failed.)
exit code 1
```

Project was created with `slidy create`, flutter_modular and mobx. I also tried with `slidy start` after creating a project with `flutter create` 
Everything is freshly installed and up-to-date (Dart, Flutter, Slidy...)

pubspec.yaml
```
name: mobx_counter
description: A new Flutter project.
publish_to: 'none' 
version: 1.0.0+1

vars: 
    clean: flutter clean
    get: flutter pub get
    runner: flutter pub run build_runner
scripts: 
    mobx_build: $runner build
    mobx_watch: $clean & $get & $runner watch
    mobx_build_clean: $clean & $get & $runner build --delete-conflicting-outputs

environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  dio: ^3.0.10
  flutter_mobx: ^1.1.0+2
  mobx: ^1.2.1+4
  flutter_modular: ^2.0.1
  flutter:
    sdk: flutter

dev_dependencies:
  mockito: ^4.1.3
  modular_codegen: ^2.1.0
  mobx_codegen: ^1.1.2
  build_runner: ^1.10.1
  flutter_test:
    sdk: flutter

flutter:
  uses-material-design: true
```

**Comment by marcelofb** [comment]
Same error :(

**Comment by FilipKvestak** [comment]
the only way (for now) is to delete version for mobx_codegen under dev_dependencies:
```
dev_dependencies:
  mockito: ^4.1.3
  modular_codegen:
  mobx_codegen: ^1.1.2
  build_runner: ^1.10.1
  flutter_test:
    sdk: flutter
```

**Comment by SidneiVerissimo** [comment]
Removing the versionfrom modular_codegen or changing it to any did the trick for me.

**Comment by fogaiht** [comment]
to fix this issue, just put modular_codegen in the last version, 2.0.1

**Comment by FilipKvestak** [comment]
So, I guess we should remove the version from modular_codegen or put it to latest?

**Comment by massilva** [comment]
> So, I guess we should remove the version from modular_codegen or put it to latest?

@FilipKvestak I fix it putting `modular_codegen` to `^2.0.0+1` version. Because `modular_codegen: ^2.0.1` didn`t create `$AppController` at `app_controller.g.dart` file.
